lib/crawler/link/checker.ex

```diff
@@ -63,4 +64,9 @@ defmodule Crawler.Link.Checker do
   defp link_path(url, base_url) do
     String.trim(url, base_url)
   end
+
+  defp strip_anchors(url) do
+    [trimmed_url | _anchors ] = String.split(url, "#")
```

Comment:
One thing just occurred to me -- I know that some single-page-apps use `/#/` for navigation instead of real paths, and this wouldn't parse those correctly. I don't know if that even matters since we don't use that pattern internally, but if you're trying to build this as a general tool that anyone could use it might be worth considering.

Reply:
I believe those would just be parsed as `/`, and throw out everything after the anchor, which is what we would want in that case. Do you have an example URL that wouldn't parse correctly?

Reply:
Or if the URL is just `href=#info`, the `trimmed_url` would be `""`. 
Then when the link lookup occurs it would use `http://[base_url] <> ""` But you're right, that could create a duplicate check